packages/flutter_tools/lib/src/base/terminal.dart

```diff
@@ -23,6 +23,17 @@ enum TerminalColor {
 AnsiTerminal get terminal {
   return context?.get<AnsiTerminal>() ?? _defaultAnsiTerminal;
 }
+
+/// Warning mark to use in stdout or stderr.
+String get warningMark {
+  return terminal.bolden(terminal.color('[!]', TerminalColor.red));
```

Comment:
How about ⚠ ?

Reply:
I tried. Unfortunately, it doesn't render on PowerShell/cmd.exe. We could detect that or leave as is.